rules-tests/CodeQuality/Rector/FuncCall/SortNamedParamRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\FuncCall\SortNamedParamRector\Fixture;
+
+function run($foo = null, $bar = null, $baz = null) {}
+
+run(bar: $bar, foo: $foo);
+
+run($foo, baz: $baz, bar: $bar);
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\FuncCall\SortNamedParamRector\Fixture;
+
+function run($foo = null, $bar = null, $baz = null) {}
+
+run(foo: $foo, bar: $bar);
+
+run($foo, bar: $bar, baz: $baz);
```

Comment:
please add test to skip first class callable

```
run(...);
```

Reply:
Isn't the check missing here too https://github.com/rectorphp/rector-downgrade-php/blob/4b311807e9aa0c8db1999c5a1adee09396f03bec/rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php#L86 ?